mantidimaging/gui/utility/qt_helpers.py

```diff
@@ -46,20 +46,20 @@ def __exit__(self, *args):
             obj.blockSignals(prev)
 
 
-def compile_ui(ui_file, qt_obj=None):
+def compile_ui(ui_file: str, qt_obj=None) -> QWidget:
     base_path = finder.ROOT_PATH
     return uic.loadUi(os.path.join(base_path, ui_file), qt_obj)
 
 
-def select_directory(field, caption):
+def select_directory(field: QWidget, caption: str) -> None:
     assert isinstance(field, QLineEdit), (f"The passed object is of type {type(field)}. This function only works with "
                                           "QLineEdit")
 
     # open file dialog and set the text if file is selected
     field.setText(QFileDialog.getExistingDirectory(caption=caption))
 
 
-def get_value_from_qwidget(widget: QWidget):
+def get_value_from_qwidget(widget: QWidget) -> Any:
```

Comment:
its a bit verbose but we could narrow this down from `Any` with 

```suggestion
def get_value_from_qwidget(widget: QWidget) -> str | float | bool | None:
```

Reply:
Might need to add `return None` at the end of the function to help out MyPy